.github/workflows/pull-with-lifecycle-manager.yaml

```diff
@@ -9,6 +9,11 @@ env:
 on:
   pull_request:
     branches: [ "main" ]
+    paths-ignore:
```

Comment:
Can you also add it on `validate crd` github action?

Reply:
Yes!